docs/docs/ProgrammingGuide/MLPipeline/MLPipeline.md

```diff
@@ -0,0 +1,109 @@
+
+## Use BigDL with Apache Spark ML pipeline ##
+
+BigDL provides `DLEstimator` and `DLClassifier` for users with Apache Spark MLlib experience, which
+provides high level API for training a BigDL Model with the Apache Spark `Estimator`/`Transfomer`
+pattern, thus users can conveniently fit BigDL into a ML pipeline.
+
```

Comment:
Add short description of DLModel and DLClassifierModel, esp. for the user to directly load a pre-trained model and use it in Spark ML for prediction.